pallets/roles/src/lib.rs

```diff
@@ -276,9 +283,15 @@ pub mod pallet {
 				Error::<T>::NoRoleAssigned
 			);
 
-			// TODO: Call jobs manager to remove the services.
-			// On successful removal of services, remove the role from the mapping.
-			// Issue link for reference : https://github.com/webb-tools/tangle/issues/292
+			// Get active jobs for the role.
+			let active_jobs = T::JobsHandler::get_active_jobs(stash_account.clone());
+			for job in active_jobs {
+				let job_key = job.0;
+				if job_key.get_role_type() == role {
+					// Submit request to exit from the known set.
+					T::JobsHandler::exit_from_known_set(stash_account.clone(), job_key, job.1)?;
```

Comment:
Can we return more useful error message from here, example lets say validator X is part of 5 jobs, and one of the jobs does not have enough validators so cannot exit, the error from here does not indicate which of those jobs is causing the issue.